src/styles.scss

```diff
@@ -1475,6 +1476,23 @@ hr {
   font-size: 12px;
 }
 
+
+.search-form-field {
+  max-width: 200px;
+}
+
+.search-form-field input {
+  padding-right: 35px;
+  transform: translateY(-3px);
+}
+
+.search-form-field button {
+  position: absolute;
```

Comment:
it's likely the absolute position is making things worse

Reply:
I added the absolute position because I found on firefox and chrome the x icons were showing up in different places